mod/wikirate/lib/stylesheets/style_wikirate_progress_bar.scss

```diff
@@ -14,6 +14,16 @@
   font-weight: bolder;
 }
 
+.progress {
+  .progress-known, .progress-unknown {
+    a {
+       color: $white-text-color;
+    }
+  }
```

Comment:
should there be something here that separates known and unknown?

Reply:
It is just white text on dark background and dark text in light background. I don't think if we need to differentiate link colours for that purpose.

Reply:
this has white for both known and unknown.  One big problem was that on staging known and unknown had the same background color.  I was asking whether there should be a general purpose fix.